File: problems/389-find-the-difference/solution.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        s_count = {}
        t_count = {}
        for char in s:
            s_count[char] = s_count.get(char, 0) + 1
        for char in t:
            t_count[char] = t_count.get(char, 0) + 1
        for char in t_count:
            if char not in s_count or t_count[char] != s_count[char]:
                return char

File: problems/389-find-the-difference/solution.c
char findTheDifference(char* s, char* t) {
    int s_sum = 0;
    int t_sum = 0;
    int i = 0;
    while (s[i] != '\0') {
        s_sum += s[i];
        i++;
    }
    i = 0;
    while (t[i] != '\0') {
        t_sum += t[i];
        i++;
    }
    return (char)(t_sum - s_sum);
}